buildspecs/build-windows.yml

```diff
@@ -0,0 +1,6 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - mvn clean install
```

Comment:
why not just build.yml?

Reply:
`-Dmaven.wagon.httpconnectionManager.maxPerRoute=2` is not working on Windows somehow. Got `Unknown lifecycle phase ".wagon.httpconnectionManager.maxPerRoute=2".` error

Reply:
I believe it's because there's no space between `-D` and the option. Adding a space should fix it

Reply:
Yeah, adding the space fixed it.